packages/notices/src/store/selectors.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_CONTEXT } from './constants';
+
+/**
+ * The default empty set of notices to return when there are no notices
+ * assigned for a given notices context. This can occur if the getNotices
+ * selector is called without a notice ever having been created for the
+ * context. A shared value is used to ensure referential equality between
+ * sequential selector calls, since otherwise `[] !== []`.
+ *
+ * @type {Array}
+ */
+const DEFAULT_NOTICES = [];
```

Comment:
Nit: Should we also move to `constants.js` file?

Reply:
Maybe. I think the main value in `constants.js` are for variable constants used across files. This one is specific to selectors, however.

Reply:
> Nit: Should we also move to `constants.js` file?

I think I'd rather keep it here, per my previous comment.